### PR TITLE
My room development

### PR DESCRIPTION
**Pull request by Devon-Brady**
### Description
  Add description here
Set up routes models needed for my room feature.
  What was changed?
A few new files were created to support the new routes and models as well as new server end points
  Why does PR exist?
To push to main
  How do your changes do what is intended?
  Will a screenshot be helpful?

### Tests
  How was it tested?
I tested the endpoints with postman and Postgres using seeded and dummy data
### Reviewers
  2 reviewers required to merge to main. Must not merge your own pull request.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ app.use(passport.session());
 
 async function verifyRole(req, res, next) {
   const userId = req.user.id;
+  console.log(req.user)
```
Comment (Plofland) [inline index.js]: We can get rid of this

File: models/my-room-model.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const database = require('../database/dbConfig');
+
+//Gets all of the rooms liked by a user
+const fetchFollowedRooms = (filter) => {
+    return database('my_room').where(filter);
+};
+
+// adds a room to a users 'my rooms'
+const add = (roomID, userID) => {
+    // TODO: handle the case where a user has already liked a room
```
Comment (Plofland) [inline models/my-room-model.js:10]: Let's add this TODO to the Trello card checklist before it gets forgotten about.

File: package.json [hunk 3]
```diff
@@ -30,7 +30,8 @@
 		"knex": "^0.21.1",
 		"passport": "^0.4.1",
 		"passport-linkedin-oauth2": "^2.0.0",
-		"pg": "^8.5.1"
+		"pg": "^8.5.1",
+		"sqlite3": "^5.0.2"
```
Comment (Plofland) [inline package.json]: We won't be needing this sqlite3 dependency for the production DB.

File: routes/my-room-router.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    .then(rooms => {
+        console.log({rooms})
+        res.status(200).json({rooms})
+    })
+    .catch((err) =>{
+        res.status(500).json(`${err}`)
+    });
+});
+
+// TODO: add a room to a users rooms
```
Comment (Plofland) [inline routes/my-room-router.js]: Get rid of the word 'TODO' unless you're still working on these endpoints.

**Review by Plofland — CHANGES_REQUESTED** [review]
After you make some very minor changes, let's get it merged in.

File: routes/my-room-router.js (hunk 4 above)
Comment (Plofland) [reply routes/my-room-router.js]: I like the comments though

File: index.js (hunk 1 above)
Comment (gvicas17) [inline index.js]: This console.log can be deleted.

**Review by gvicas17 — DISMISSED** [review]
Besides getting rid of that console.log it looks good!

**Review by Plofland — APPROVED** [review]
Thanks for making the changes. With the exception of the one console.log that Gabby found, it looks good to me.

**Review by dustyfingers — APPROVED** [review]

**Review by dustyfingers — APPROVED** [review]

**Review by ialkamal — APPROVED** [review]
Looks good. reviewed with Devon.

**Review by gvicas17 — APPROVED** [review]
Looks good!

**Pull request merged**